src/backend/controllers/tankController.js

```diff
@@ -1,68 +1,64 @@
 // @flow strict
 
-// Flow type import
-import type { Request, Response } from 'express';
-
 const User = require('../../models/userModel');
 const Tank = require('../../models/tankModel');
 
 const { validationResult } = require('express-validator');
 
-exports.getFavorite = async (req: Request, res: Response) => {
+// Flow type import
+import type {
+	$Request,
+	$Response,
+	NextFunction,
+	Middleware,
+} from 'express';
+
+exports.getFavorite = async (req: $Request, res: $Response) => {
 	await User.findById(req.user.id, 'favoriteTankId', (err: Error, myUser: User) => {
 		if(err) {
 			res.send(err);
 			console.log('could not find user');
 		} 
-		else {
+		else
 			res.send(myUser.favoriteTankId);
-			console.log('Retrieved Favorite Tank.');
-		}
 	});
 }
 
-exports.favoriteTank = async (req: Request, res: Response) => {
+exports.favoriteTank = async (req: $Request, res: $Response) => {
 	// the 'new' option means return the document after it has been updated
-	await User.findOneAndUpdate( { _id: req.user.id }, {favoriteTankId : req.body.favoriteTankId}, {'new':true }, (err: Error, result: User) => {
+	// the 'useFindAndModify' is just so it doesnt throw a deprecation warning
+	await User.findOneAndUpdate( { _id: req.user.id }, {favoriteTank : req.body.favoriteTankId}, {'new':true, 'useFindAndModify':false}, (err: Error, result: User) => {
 		if(err) {
 			res.status(500).send(err);
 			console.error(err);
 		}
-		else {
-			res.send(result.favoriteTankId);
-			console.log('Updated favorite tank.');
```

Comment:
I guess this is more of a question of whether it's right to make logs for successful API calls.

Some controllers are using `return res.status(200).json({ msg: 'Success message here' });` or
```
console.log('Success message here');
return res.status(200).json(dbObject);
```
What makes these an exception to not including some log going out to confirm a route worked as it should?

Reply:
I have no strong feelings either way. I'm a little bit partial to successful console.logs() because that way when something inevitably _doesn't_ work, we can see that it is the front-end's fault/a connection issue more easily.

We should probably pick something and be consistent with it though.